recipe/meta.yaml

```
@@ -1,17 +1,17 @@
 {% set name = "conda-index" %}
-{% set version = "0.5.0" %}
+{% set version = "0.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 12d78ec6dc55f64ad68761108a416ebde3dc6472f257045ddfa45378980c9074
+  sha256: b096e34b048b494489ff41389ecc4b17b9f400dbf5b36ee2e39aa2558cc9a6c2
 
 build:
   number: 0
-  skip: True  # [py<38]
+  skip: true  # [py<38]
```

```suggestion
  skip: true  # [py<38 or py>312]
```
Since `conda-index` requires `conda-build >=24.1.0` to run (see [check](https://github.com/conda/conda-index/blob/0.6.0/conda_index/plugin.py#L22)) and `conda-build` hasn't been built for python 3.13 yet, doesn't it make more sense to just not build `conda-index` for 3.13? Is there a use in having it available as a library but not a runable conda command?

---

It requires conda-build to test; and if conda-build is installed and less than a certain version then it will hide the conda plugin.

If conda-build is newer (doesn't provide its own "conda index") or if conda-build is uninstalled, then "conda index" will show up.

`python -m conda_index` is also supported.

---

Ah, I forgot about the module runner. LGTM